src/database/migrations/create-7-comments.js

```diff
@@ -10,7 +10,7 @@ export default {
       type: Sequelize.STRING,
       allowNull: false
     },
-    requestId: {
+    reqId: {
```

Comment:
You should change this back to requestId. It is much more understandable than reqId

Reply:
I alright. I guess that happened during rebasing. I'll fix it.